### PR TITLE
Fix 250 -  Providing the Selected Background Wallpaper Preview

### LINKED ISSUE #250: Providing the Selected Background Wallpaper Preview
**I'm submitting a ...**  
- [x] feature request

**Actual Behaviour**
The user can select the image but there should be a preview for it.

screenshot from 2018-10-05 20-59-00

**Expected Behaviour**
Image Preview should be Present.

**Other information:**

**System information:**

```
Your operating system
```
Ubunutu

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Providing the Selected Background Wallpaper Preview

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
ezgif com-video-to-gif 10

- Providing Image Preview

Fixes #250

### CODE REVIEW

**Comment by xeon-zolt** [comment]
@sk9331657 their is a merge conflict

**Comment by sk9331657** [comment]
@xeon-zolt  Actually I formatted document to give proper indentation.

**Comment by sk9331657** [comment]
@ankurg22  Please review it.

**Comment by sk9331657** [comment]
@r0mflip Please review. I have made  suggested Changes

**Comment by sk9331657** [comment]
@mariobehling  @abishekvashok  Please merge it.

**Comment by sk9331657** [comment]
@r0mflip  Please review it.

**Comment by r0mflip** [comment]
> @xeon-zolt Actually I formatted document to give proper indentation.

That can be done in a separate PR. IDK but we should actually have a style guide.

> @r0mflip Please review. I have made suggested Changes

LGTM

**Comment by meets2tarun** [comment]
>we should actually have a style guide.

@r0mflip We actually have it [here](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/).

File: templates/index.html [hunk 1]
```diff
@@ -50,450 +50,482 @@
...
+            </a>
+          </div>
+          <div class="custom-menu-item">
+            <a href="https://pslab.fossasia.org" target="_blank">
+              <div class="custom-icon"><img src="{{ url_for('static', filename='Pslab.png') }}"></div>
+              <p class="custom-title">PS Lab</p>
+            </a>
+          </div>
+          <hr style="margin: 10px">
+          <div style="display: flex;justify-content: center;align-items: center; margin: 0 0 -20px 0">
```
Comment (tabesin) [inline templates/index.html]: Is it really necessary to put style in the html template? try to use CSS for MVC compliance.

File: templates/index.html [hunk 2]
```diff
@@ -50,450 +50,482 @@
...
+            More on&nbsp;<a href="https://labs.fossasia.org/" target="_blank" style="text-decoration: none;color: #737373">
```
Comment (tabesin) [inline templates/index.html]: same

Comment (tabesin) [reply templates/index.html]: I opened a new ticket to resolve inline CSS issues.

Comment (sk9331657) [reply templates/index.html]: Ok

**Pull request merged**